services/src/main/resources/keycloak-default-client-types.json

```diff
@@ -11,6 +11,65 @@
         }
       }
     },
+    {
+      "name": "oidc",
+      "provider": "default",
+      "config": {
+        "authorizationServicesEnabled": {
+          "applicable": true,
+          "default-value": false
+        },
+        "consentRequired": {
+          "applicable": true,
+          "default-value": false
+        },
+        "directAccessGrantsEnabled": {
+          "applicable": true,
+          "default-value": false
+        },
+        "frontchannelLogout": {
+          "applicable": true,
+          "default-value": false
+        },
+        "fullScopeAllowed": {
+          "applicable": true,
+          "default-value": true
+        },
+        "implicitFlowEnabled": {
+          "applicable": true,
+          "default-value": false
+        },
+        "nodeReRegistrationTimeout": {
+          "applicable": true,
+          "default-value": -1
+        },
+        "oauth2.device.authorization.grant.enabled": {
+          "applicable": true,
+          "default-value": "false"
+        },
+        "oidc.ciba.grant.enabled": {
+          "applicable": true,
+          "default-value": "false"
+        },
+        "protocol": {
+          "applicable": true,
+          "default-value": "openid-connect",
+          "read-only": true
+        },
+        "publicClient": {
+          "applicable": true,
+          "default-value": true
+        },
+        "serviceAccountsEnabled": {
+          "applicable": true,
+          "default-value": false
+        },
+        "standardFlowEnabled": {
```

Comment:
What do you think of having these auth flows defaulted to false but not read-only? I could see a case where one would want to pick a generic OIDC client type and specify things themselves, but if this to be a base for other client types, it might also make sense to make these read-only and have the children client types override the fields.